src/js/tabs/receiver.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const CryptoJS = require("crypto-js");
```

Comment:
Would like to avoid using `require` if possible, but there is tail of other stuff which would be needed so just let's keep it here for now.